### DISCUSSION TITLE
Welcome to FLTK Discussions!

### DISCUSSION THREAD

**Discussion by Albrecht-S** [fltk/fltk#998] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Albrecht-S** [comment]
**_Note:_** Using GitHub Discussions for FLTK is currently experimental (as of July 2024).

If it turns out to be useful and usable we may decide to replace our Google Groups ([fltk.general](https://groups.google.com/g/fltkgeneral) and [fltk.coredev](https://groups.google.com/g/fltkcoredev)) with GitHub discussions in the future.

FYI: For those that can't access Google (Groups) - from _"some parts of the world"_ - using GitHub Discussions can provide an alternative.

**Comment by erco77** [comment]
OK, hmm, I need to make some changes to the backend scripts to handle these messages, as currently they're going directly to my personal account because it's a new message type not handled by the email redirector.

**Comment by erco77** [comment]
OK, created the fltk.discussions _read-only_ NNTP newsgroup to keep track of the github discussions, so these messages have a place to go.

Should be visible here:
- Web: https://www.fltk.org/newsgroups.php?gfltk.discussions+T
- NNTP: [news://fltk.org:1024/fltk.discussions](news://fltk.org:1024/fltk.discussions)

**Comment by dannye** [comment]
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

:wave: Hi! I'm dannye and I work on a [Game Boy song editor](https://github.com/fltk/fltk/discussions/1042) built with FLTK. I've also reported [handfuls of bugs](https://github.com/fltk/fltk/issues?q=is%3Aissue+is%3Aclosed+author%3Adannye) that I've discovered along the way. And I've even recently made my [first contribution](https://github.com/fltk/fltk/commit/46dd1b33ccb1e34be0b7a6d41a162e4dd611a548) to FLTK myself. I love this project and am glad to be here :smiley:
